### ISSUE TITLE
Accessibility finding: Text cut off between 769 - 1300 px on Form view page

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#139075]
### Checklist item

ADE-004 Font size can be set to "Very Large" or equivalent without breaking the page

### Severity level

2, High. Should be fixed in 1-2 sprints post-launch.

### Page(s)/url(s)

https://staging.va.gov/my-va/submissions/9093f30e-0d09-43ff-a13f-087847063071

### Details

Form View page: When the font is set to "Very Large" and the browser is set to anywhere between 769px and 1300px, the text is cut off on the left side. This only occurs on the form viewer pages, NOT the "We can't match your records" page.

Example: the browser is set to 769 pixels, note how the text is cut off:

<img width="559" height="429" alt="Image" src="https://github.com/user-attachments/assets/0f192817-c4cc-4377-9eb8-84e9795c08f6" />

### Reproduction steps

Log in as user 46, then go to the page. In Chrome, use the Settings to change font size to "Very Large." Set the browser width to 769px.

### Proposed solution or next steps

Please review the CSS for issues related to padding or margins on the left and right of the content wrapper.
